<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartDlg</class>
 <widget class="QDialog" name="DepartDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Departments</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wdEdit" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbPlus">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="resource1.qrc">
          <normaloff>:/some/ico/ico/plus.png</normaloff>:/some/ico/ico/plus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbProps">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Properties</string>
        </property>
        <property name="icon">
         <iconset resource="resource1.qrc">
          <normaloff>:/some/ico/ico/edit.png</normaloff>:/some/ico/ico/edit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbMinus">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="resource1.qrc">
          <normaloff>:/some/ico/ico/minus.png</normaloff>:/some/ico/ico/minus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="twDepart">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbPlus</sender>
   <signal>clicked()</signal>
   <receiver>actionNew</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbProps</sender>
   <signal>clicked()</signal>
   <receiver>actionRename</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMinus</sender>
   <signal>clicked()</signal>
   <receiver>actionDelete</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
